docs/src/main/sphinx/connector/elasticsearch.rst

```diff
@@ -390,7 +390,11 @@ Property name                                    Description
 ``elasticsearch.aws.iam-role``                   Optional ARN of an IAM Role to assume to connect to the Elasticsearch domain.
                                                  Note: the configured IAM user has to be able to assume this role.
 
+``elasticsearch.aws.iam-session-name``           Optional session name to use while assuming AWS IAM role.
+
 ``elasticsearch.aws.external-id``                Optional external ID to pass while assuming an AWS IAM Role.
+
+``elasticsearch.aws.sts-endpoint``               Optional STS endpoint to connect to while assuming AWS IAM Role.
```

Comment:
Shouldn't this be in the ES commit?

Reply:
Rebased and rearranged commits to do just that.